### ISSUE TITLE
Question about monitoring howto100m pre-training

### ISSUE THREAD

**Issue by Flaick** [facebookresearch/ProcedureVRL#7]
Hello, sorry to bother you a lot times but I still have some question about the codebase. So I am pre-training the model from scratch using howto100m subset, and I find that the **'label**' from dataloader is actually for the task recognition and the Top-1&5 error are not representing the real training curve. The only monitoring way is the loss. Is that correct?

Also when I run the code using '**procedurevrl_sgd.yaml'**, the code goes wrong, saying the preds.shape[0] is not match to labels.shape[0], and I am confused because this is actually useless if my hypothesis in previous paragraph is correct.

Thank you for your effort, I hope to receive the message from you soon.

**Comment by YiwuZhong** [comment]
@Flaick, we didn't use `label` during pre-training and thus the loss is the only metric to observe the training process. It should be straightforward to fix the bug of shape mismatch. Will update it in next version.

**Comment by Flaick** [comment]
Thank you for the response, I am wondering if you can share the log file to a better reproduction, thanks!

**Comment by YiwuZhong** [comment]
Hi @Flaick, the log file is not available now. The information I can provide is that, the loss was reduced to around 3.6-3.7 at the end of pre-training on full HowTo100M dataset.

**Comment by Flaick** [comment]
Thanks for the reponse, that's helpful

**Comment by Flaick** [comment]
Btw, do you happen to know the resolution of the video of Howto100m, I just found my resolution is around 320p. Thanks in advance!

**Comment by YiwuZhong** [comment]
@Flaick, the resolution of Howto100m videos depends on the script you wrote to download them. No matter what resolution the videos have, the data preprocessing pipeline in dataloader will compress them into the same resolution.
